dev/com.ibm.ws.logging_2_fat/fat/src/com/ibm/ws/logging/fat/TimeBasedLogRolloverTest.java

```diff
@@ -306,9 +310,7 @@ public void testEnableDisableTimedRollover() throws Exception {
         serverInUse.restoreServerConfiguration(); //restore disabled server config
         serverInUse.waitForStringInLogUsingMark("CWWKG0017I"); //wait for server config update
 
-        long nextRollover = cal.getTimeInMillis() - Calendar.getInstance().getTimeInMillis();
-        if (nextRollover > 0)
-            Thread.sleep(nextRollover + FILE_WAIT_SECONDS_PADDING); //sleep for another length of the old specified log rolloverInterval
+        Thread.sleep(10000);
```

Comment:
Is this sleep for 10 secs needed, why was the above ` long nextRollover = cal.getTimeInMillis() - Calendar.getInstance().getTimeInMillis();` and if condition removed? I think we shouldn't explicitly use sleep here, maybe do a retry instead, if needed?